tests/cluster/functions/test_drop_table_cascade.py

```diff
@@ -1,81 +0,0 @@
-import unittest
```

Comment:
should we still have the possibility for others to use this function?
It is not a function that exists in databricks, so maybe we could still have it even though we dont use it 1:1 in the TableConfigurator-setup.

Reply:
only tests were removed